Automattic/wp-calypso
# A4A: Update Product listing to follow design specs.

This pull request aligns the A4A Marketplace design specs with the current product listing. Existing components were tweaked to match the latest design.

https://github.com/Automattic/wp-calypso/assets/56598660/a37f6753-cfab-4df7-9d41-cfcf4eb9fe9f


Closes https://github.com/Automattic/jetpack-genesis/issues/285

## Proposed Changes

* Jetpack Manage's `LicenseProductCard` and `LicenseMultiProductCard` components have been ported and updated to use a button instead of a checkbox to represent the selection state, following the design specification.
* The IssueLicense component was renamed to the ProductsOverview component. Sub-components and CSS classes were also updated accordingly to match the new component name.
* Product listing sections have been updated; `<hr/>` removed, Product Icons (Jetpack & Woo) added to title.

## Testing Instructions

* Switch branch: `git checkout update/a4a/marketplace-product-card-design`.
* Start the server by running `yarn start-a8c-for-agencies`.
* Use the live link and go to`/marketplace`
* Confirm that the product overview page has been updated correctly based on the new design.
* **Note that the volume pricing has not been included in this PR and will be addressed in a separate PR.**

## Pre-merge Checklist
